src/envdrift/scanner/engine.py

```
@@ -174,21 +208,36 @@ def scan(self, paths: list[Path]) -> AggregatedScanResult:
+                    self.config.include_git_history
+                ): scanner
+                for scanner in self.scanners
+            }
+            
+            # Collect results as they complete
+            for future in as_completed(future_to_scanner):
+                scanner = future_to_scanner[future]
+                try:
+                    result = future.result(timeout=600)  # 10 minute per-scanner timeout
+                    results.append(result)
+                except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)